projects/iio_demo/src/parameters.h

```diff
@@ -43,12 +43,21 @@
 /***************************** Include Files **********************************/
 /******************************************************************************/
 
+#ifdef XILINX_PLATFORM
 #include <xparameters.h>
+#endif
+
+#ifdef ADUCM_PLATFORM
+#include "irq_extra.h"
+#endif
 
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
 
+
+#ifdef XILINX_PLATFORM
```

Comment:
I am not sure we need an `#ifdef XILINX_PLATFORM` here if we already check the `_XPARAMETERS_PS_H_` define. `_XPARAMETERS_PS_H_` should exist only on Xilinx PS platforms.

Reply:
I use XILINX_PLATFORM because it is defined in app_config.h